validphys2/src/validphys/closuretest/multiclosure_pseudodata.py

```diff
@@ -31,10 +31,10 @@ def fits_dataset_cvs(fits_dataset):
         # and then cut the central values manually.
         # TODO: Save central values in nice table like pseudodata
         # but this should be done beyond NNPDF4.0
-        cd_df = pd.read_csv(ds.commondata.datafile, sep=r'\s+', skiprows=1, header=None)
+        cd_df = ds.commondata.metadata.load_data_central()
```

Comment:
I don't think this will do what you actually want.
I don't like this function at all _but_ to recover the old behaviour you probably want to do `ds.commondata.with_modified_data(the fit data path)`

Reply:
But to be honest, why don't you just get in the input the data loaded using use_fit_commondata.

This function looks like a convenient way of loading the data to avoid running over fits but I think it might have been hiding errors for a while...